tests/console/docker_compose.pm

```diff
@@ -30,7 +30,8 @@ use strict;
 use warnings;
 
 sub run {
-    select_console("root-console");
+    my ($self) = @_;
```

Comment:
since we excluding it for SLE . does it make sense to touch it ? do we plan also migrate to `VIRTIO_CONSOLE=1` in o3 ?

Reply:
We might include it in the future and all these tests are a common subset of container tests, which I think should be similar the way they are handled.

Reply:
Is there any reason why not? BTW where are these tests in o3?

Reply:
A test called `extra_tests_textmode_containers` in Tumbleweed builds e.g. https://openqa.opensuse.org/tests/1282756
and Leap https://openqa.opensuse.org/tests/1282504#